.scripts/mark_file/main.py

```diff
@@ -0,0 +1,111 @@
+import polib
+import os
+import datetime
+import requests
+
+
+def entry_check(pofile):
+    lines_tranlated = len(pofile.translated_entries())
+    lines_untranlated = len(pofile.untranslated_entries())
+
+    if lines_tranlated == 0:
+        result = "❌"
+    elif lines_untranlated == 0:
+        result = "✅"
+    else:
+        lines_all = lines_tranlated + lines_untranlated
+        progress = lines_tranlated / lines_all
+        progress = round(progress*100, 2)
+        result = f"Ongoing, {str(progress)} %"
+
+    return result
+
+
+def get_github_issue():
+    NUMBER_OF_ISSUES = 100
+
+    url = f"https://api.github.com/repos/python/python-docs-zh-tw/issues?per_page={NUMBER_OF_ISSUES}"
+    headers = {
+        "Accept": "application/vnd.github+json",
+        "X-GitHub-Api-Version": "2022-11-28"
+    }
+    r = requests.get(url=url, headers=headers)
+    result = r.json()
+
+    result_list = []
+    for issue in result:
+        title = issue['title'].split(" ")
+
+        if len(title) < 2:
+            continue
+        if title[0] != "翻譯" and title[0].lower() != "translate":
+            continue
+        if issue["assignee"] is None:
+            continue
+
+        filename = title[1]
+        if filename[0] == "`":
+            filename = filename[1:]
+        if filename[-1] == "`":
+            filename = filename[:-1]
+
+        filename = filename.split("/")
```

Comment:
如果 variable 是個 list/array，naming 通常會是複數型，提高可讀性
```suggestion
        filename_segments = filename.split("/")
```